components/count-badge/test/count-badge-icon.axe.js

```diff
@@ -0,0 +1,24 @@
+import '../count-badge-icon.js';
+import { expect, fixture, html } from '@open-wc/testing';
+
+const countBadgeTooltip = html`
```

Comment:
For each of the badges, I've added cases for the tooltip, announce-changes, and a basic badge since they have slightly different accessibility labels.